src/doc/python_scripting/functions.rst

```diff
@@ -1510,38 +1510,52 @@ expression : string
     The expression definition as a string.
 
 return type : CLI_return_t
-    The DefineExpression functions return 1 on success and 0 on failure.
+    The DefineArrayExpression function returns 1 on success and 0 on failure.
 
 
 **Description:**
 
     DefineArrayExpression creates new array variables.
-    Expression variables can be plotted like any other variable.
-    The variableName argument is a string that contains the name of the new
-    variable. You can pass the name of an existing expression if you want
-    to provide a new expression definition.
-    The expression argument is a string that contains the definition of the
-    new variable in terms of math operators and pre-existing variable names
-    Reference the VisIt User's Manual if you want more information on
-    creating expressions, such as expression syntax, or a list of built-in
-    expression functions.
+    Array variables are a collection of scalar variables that are grouped together.
+    All the variables must have the same centering and only scalar variables are supported, for example, no vector, tensor or material variables.
+    Array variables are used in the Label plot.
+
+    The variableName argument is a string that contains the name of the new variable.
+    You can pass the name of an existing expression if you want to provide a new expression definition.
+    The expression argument is a string that contains the definition of the new variable in terms of :ref:`built-in expressions <Built-in_expressions>` and pre-existing variable names using VisIt_'s :ref:`expression grammar <Expression_grammar>`.
+    If you run into problems defining your expression you might want to read the section on :ref:`expression compatibility gotchas <Expression_Compatibility_Gotchas>`.
 
 
 **Example:**
 
 ::
 
   #% visit -cli
-  OpenDatabase("/usr/gapps/visit/data/globe.silo")
-  DefineScalarExpression("myvar", "sin(u) + cos(w)")
-  # Plot the scalar expression variable.
-  AddPlot("Pseudocolor", "myvar")
-  DrawPlots()
-  # Plot a vector expression variable.
-  DefineVectorExpression("myvec", "{u,v,w}")
-  AddPlot("Vector", "myvec")
-  DrawPlots()
-
+  OpenDatabase("/usr/gapps/visit/data/curv3d.silo")
+  DefineScalarExpression("d1", 'recenter(d, "zonal")')
+  DefineScalarExpression("p1", 'recenter(p, "zonal")')
+  # Define an 2 array variables from 2 scalars. In both
```

Comment:
```suggestion
  # Define 2 array variables, each from 2 scalars. Here, we
  # reuse the same scalars twice for illustrative purposes
  # only. Normally, the scalars are different.
```

Reply:
Proposed above as a tad clearer and then elim 1538-1540 (which I can't easily do in a simle *suggestion* like this. Yet another reason that where possible (I know its not appropriate in this context), sentence per line is a little friendlier to dev tools.